### DISCUSSION TITLE
Stats card not working properly

### DISCUSSION THREAD

**Discussion by FRJess** [DenverCoder1/github-readme-streak-stats#610] [General]
Hi all!

It's been several days since my stats card is not working properly and does not appear on my READme. I have checked all the settings and cannot find the issue.

Is there any issue with the project?

```
 <a href="https://git.io/streak-stats">
      <img src="https://streak-stats.demolab.com?user=frjess&theme=synthwave&date_format=j%20M%5B%20Y%5D" alt="frjess" /
  </a>
```

EDIT: Issue resolved

**Comment by ghost** [comment]
Hi @FRJess 
How were you able to resolve the issue?
